### PR TITLE
[#517] Create projects with main branch

### LINKED ISSUE #517: Create projects with main branch
Move to the `main` branch

### PR DESCRIPTION
**Pull request by vrom911**
Resolves #517

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -789,8 +789,8 @@ If you'd like to take part in the development processes, here are a few things t
 
 * Summoner is a multi-package project which has two packages inside: `summoner-cli` and `summoner-tui`. We are supporting the same functionality in both of them, so, if it's possible, you should add/change the code in both of the packages.
 * Summoner uses the custom prelude [`relude`](https://github.com/kowainik/relude).
-* We are using the [Kowainik style guide](https://github.com/kowainik/org/blob/master/style-guide.md#haskell-style-guide).
-* To sum up, [here is the Contributing guide](https://github.com/kowainik/.github/blob/master/CONTRIBUTING.md#contributing-to-the-kowainik-repositories) we use across the repositories.
+* We are using the [Kowainik style guide](https://github.com/kowainik/org/blob/main/style-guide.md#haskell-style-guide).
```
Comment (chshersh) [inline README.md:792]: I just noticed, that this could be a link to our blog post, not the markdown file 🙂

File: summoner-cli/src/Summoner/Project.hs [hunk 2]
```diff
@@ -296,7 +296,9 @@ doGithubCommands Settings{..} = do
             Just _ -> do
                 isHubSuccess <- runHub repo
                 if isHubSuccess
-                then "git" ["push", "-u", "origin", "master"]
+                then do
+                    "git" ["push", "-u", "origin", "main"]
```
Comment (chshersh) [inline summoner-cli/src/Summoner/Project.hs:300]: I just googled and I see that the `git init` command now takes the argument `--initial-branch` to specify the default branch since `git 2.28`. I guess, we can add this option to our `git init` command above and update the README with the minimal `git` version required for the tool to work.

* Source: https://stackoverflow.com/questions/42871542/how-can-i-create-a-git-repository-with-the-default-branch-name-other-than-maste

**Review by chshersh — COMMENTED** [review]
Great work! I left one comment with one more patch we need to do, I guess. But overall looks great 👍🏻 Gigantic work! 🦶🏻

**Review by chshersh — APPROVED** [review]
Superb! 👍🏻
